src/components/NewWorkspaceModal.test.ts

```diff
@@ -239,19 +266,29 @@ describe('NewWorkspaceModal', () => {
   });
 
   it.each([
-    { cloudPlatform: 'AZURE', expectedBillingProjects: ['Azure Billing Project'] },
-    { cloudPlatform: 'GCP', expectedBillingProjects: ['Google Billing Project'] },
-  ] as { cloudPlatform: CloudPlatform; expectedBillingProjects: string[] }[])(
-    'can limit billing projects to one cloud platform',
-    async ({ cloudPlatform, expectedBillingProjects }) => {
+    {
+      cloudPlatform: 'AZURE',
+      expectedBillingProjects: ['Azure Billing Project', 'Protected Azure Billing Project'],
+      requireEnhancedBucketLogging: false,
+    },
+    {
+      cloudPlatform: 'AZURE',
+      expectedBillingProjects: ['Protected Azure Billing Project'],
+      requireEnhancedBucketLogging: true,
+    },
+    { cloudPlatform: 'GCP', expectedBillingProjects: ['Google Billing Project'], requireEnhancedBucketLogging: false },
+    { cloudPlatform: 'GCP', expectedBillingProjects: ['Google Billing Project'], requireEnhancedBucketLogging: true },
```

Comment:
`requireEnhancedBucketLogging` doesn't impact which GCP billing projects are presented-- it is used instead to force the checkbox for additional security/logging.